### PR TITLE
update to Ubuntu 18.04

### PR DESCRIPTION
**Pull request by KrisThielemans**

### CODE REVIEW

**Comment by paskino** [comment]
[Vagrant](https://www.vagrantup.com/docs/boxes.html#official-boxes) suggests to use [bento](https://app.vagrantup.com/bento) boxes as base boxes for different linux flavours.

We could/should change to [bento/ubuntu-18.04](https://app.vagrantup.com/bento/boxes/ubuntu-18.04).

**Comment by paskino** [comment]
That error is due to the name of the `"--storagectl", "IDE"` which should be `IDE Controller`

**Comment by paskino** [comment]
This branch creates a VM that works fine with [ubuntu18.04](https://github.com/CCPPETMR/SIRF-SuperBuild/tree/ubuntu18.04) SuperBuild branch. 

Tests and notebooks (running on VM and accessed from host) work.

Requires https://github.com/CCPPETMR/SIRF-SuperBuild/pull/149

**Comment by paskino** [comment]
https://github.com/CCPPETMR/SIRF-SuperBuild/pull/149 is superseded by 
https://github.com/CCPPETMR/SIRF-SuperBuild/pull/205 (merged)

**Comment by paskino** [comment]
closes #94

**Comment by paskino** [comment]
probably needs https://github.com/CCPPETMR/SIRF-SuperBuild/pull/220

The VM gets created correctly on the ubuntu18.04 branch of the SuperBuild

File: scripts/bootstrap_ubuntu.sh [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+fi
+cd $userHOME/devel
+
+if [ ! -d $userHOME/devel/CCPPETMR_VM ]; then
+  git clone https://github.com/CCPPETMR/CCPPETMR_VM.git
+  cd CCPPETMR_VM
+  git checkout ubuntu18.04
+else
+  cd CCPPETMR_VM
+  git checkout ubuntu18.04
```
Comment (KrisThielemans) [inline scripts/bootstrap_ubuntu.sh]: will need to change

File: scripts/bootstrap_ubuntu.sh [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+if [ ! -d $userHOME/devel ]; then
+  mkdir $userHOME/devel 
+fi
+cd $userHOME/devel
+
+if [ ! -d $userHOME/devel/CCPPETMR_VM ]; then
+  git clone https://github.com/CCPPETMR/CCPPETMR_VM.git
+  cd CCPPETMR_VM
+  git checkout ubuntu18.04
```
Comment (KrisThielemans) [inline scripts/bootstrap_ubuntu.sh]: will need to change

File: scripts/bootstrap_ubuntu.sh [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+bash $userHOME/devel/CCPPETMR_VM/scripts/INSTALL_CMake.sh /usr/local
+bash $userHOME/devel/CCPPETMR_VM/scripts/INSTALL_python_packages.sh
+
+# port 8888 is forwarded to 8888 so the user can use the host browser
+# Therefore no browser is installed on the VM
+python -m pip install jupyter
+python -m pip install spyder
+
+chown -R $SIRFUSERNAME:users $userHOME
+# the ubuntu18.04 branch latest commit
```
Comment (KrisThielemans) [inline scripts/bootstrap_ubuntu.sh]: delete

File: vagrant/Vagrantfile [hunk 4]
```diff
@@ -62,17 +65,17 @@ Vagrant.configure("2") do |config|
...
-       "--storagectl", "IDE",
+       "--storagectl", "IDE Controller",
         "--port", "1",
        "--device", "0",
        "--type", "dvddrive",
-       "--medium", "emptydrive",
+       "--medium", "emptydrive"
        ]
-	vb.name = "SIRF_1.1.1"
+	vb.name = "SIRFCIL_2.0_18.04"
```
Comment (KrisThielemans) [inline vagrant/Vagrantfile]: remove CIL

File: CHANGES.md [hunk 5]
```diff
@@ -1,5 +1,13 @@
 # ChangeLog
 
+## 2.0rc1
+- upgraded to Ubuntu 18.04
+- add port forwarding (8888 -> 8888 for jupyter)
```
Comment (KrisThielemans) [inline CHANGES.md]: and gadgetron?

**Comment by KrisThielemans** [comment]
will need to update `CHANGES.md`

File: CHANGES.md (hunk 5 above)
Comment (paskino) [reply CHANGES.md]: Do we have to document it also here?

Comment (paskino) [reply CHANGES.md]: done it

**Pull request merged**